components/resources/library/src/desktopMain/kotlin/org/jetbrains/compose/resources/ResourceEnvironment.desktop.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.compose.resources
+
+import org.jetbrains.skiko.SystemTheme
+import org.jetbrains.skiko.currentSystemTheme
+import java.awt.Toolkit
+import java.util.*
+
+internal actual fun getResourceEnvironment(): ResourceEnvironment {
+    val locale = Locale.getDefault()
+    //FIXME: don't use skiko internals
+    val isDarkTheme = currentSystemTheme == SystemTheme.DARK
```

Comment:
There is `isSystemInDarkTheme`, but it requires composable context

Reply:
what do you want to say?